apps/web/src/app/globals.css

```
@@ -596,10 +602,12 @@
 			border-shape: shape(
 				from 0 0,
 				hline to 100%,
-				curve to calc(100% - 56px) 100% with
-					calc(100% - 28px) 0 / calc(100% - 28px) 100%,
-				hline to 56px,
-				curve to 0 0 with 28px 100% / 28px 0,
+				curve to calc(100% - var(--notch-sweep)) 100% with
+					calc(100% - var(--notch-sweep) / 2) 0 /
```

Stylelint (17.14.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)